UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -228,20 +241,28 @@ public void ServerPerformInteraction(HandApply interaction)
 				{
 					if (channel == StatusDisplayChannel.DoorTimer)
 					{
-						currentTimerSeconds += 60;
-						if (currentTimerSeconds > 600)
+						if (!AccessRestrictions || AccessRestrictions.CheckAccess(interaction.Performer))
 						{
-							currentTimerSeconds = 1;
-						}
-
-						if (!countingDown)
-						{
-							StartCoroutine(TickTimer());
+							currentTimerSeconds += 60;
+							if (currentTimerSeconds > 600)
+							{
+								currentTimerSeconds = 1;
+							}
+
+							if (!countingDown)
+							{
+								StartCoroutine(TickTimer());
+							}
+							else
+							{
+								OnTextBroadcastReceived(StatusDisplayChannel.DoorTimer);
+							}
 						}
 						else
 						{
-							OnTextBroadcastReceived(StatusDisplayChannel.DoorTimer);
+							Chat.AddExamineMsg(interaction.Performer, $"Access Denied.");
```

Comment:
This is a just a suggestion, could maybe also a play a sound here.

Reply:
I'll see about finding the most annoying sound. Bad enough to give reason for officers to increase the time.